<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boolean Functions Solver</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-Boolean-Functions-Solver-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/debug/Icono.jpeg</normaloff>../build-Boolean-Functions-Solver-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/debug/Icono.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>521</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Analizador</string>
     </attribute>
     <widget class="QPushButton" name="pushButton33">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ab'cd'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton32">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>abcd'</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxD">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>390</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton01">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'bc'd'</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxB">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditFuncion">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton20">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'b'cd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton30">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'b'cd'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton12">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>abc'd</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxA">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonBorrar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxC">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton13">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ab'c'd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'b'c'd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton03">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ab'c'd'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton00">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'b'c'd'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton31">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'bcd'</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton02">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>abc'd'</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>c:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton11">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'bc'd</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditResult">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="overwriteMode">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonEvaluar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>350</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Evaluar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton22">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>abcd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton21">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>a'bcd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>f(a,b,c,d) =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>390</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>d:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton23">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>ab'cd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>b:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Bitácora</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>481</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Fecha</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Funcion</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
